react/src/lib/Lightbox/index.js

```diff
@@ -581,6 +583,10 @@ Lightbox.defaultProps = {
   onChange: null,
   onClose: null,
   onDownload: null,
+  popoverProps:{
+    isContained:true,
+    direction: 'bottom-left'
```

Comment:
Why are we changing the default from bottom right to bottom left?

Reply:
fixed @holsted